### PR TITLE
Used react-toastify for new notifications

### PR DESCRIPTION
**Pull request by tmenezes1**
### Summary 

Submitting a review now results in either a success or error toast notification, as opposed to alert popups that were used previously.

image
image

### Test Plan 

Tested on successful course review submissions

### CODE REVIEW

**Review by khemritolya — CHANGES_REQUESTED** [review]
Looks good overall -- there were some style issues however -- see if you can move it to the center of the screen, and make it larger. In user testing it seems that people miss the popup sometimes if it is in the corner.

**Review by juliejeong — COMMENTED** [review]
Looks good!! the color is bright so it is easy to catch attention and also the duration of the pop up seems reasonable as well :)

File: client/src/ui/Form.jsx [hunk 1]
```diff
@@ -248,11 +262,14 @@ export default class Form extends Component {
         Session.setPersistent({ "review_num": "" });
         Session.setPersistent({ "courseId": "" });
         this.hide();
-        alert('Thanks for reviewing! New reviews are updated every 24 hours.');
+        //alert('Thanks for reviewing! New reviews are updated every 24 hours.');
```
Comment (khemritolya) [inline client/src/ui/Form.jsx:265]: I'm not sure why these comments are still hanging around. They appear in a number of other places as well. It is recommended to get rid of unneeded code.

File: client/src/ui/Form.jsx [hunk 2]
```diff
@@ -248,11 +262,14 @@ export default class Form extends Component {
...
+        //toast('Thanks for reviewing! New reviews are updated every 24 hours.')
+        this.createNotification('Thanks for reviewing! New reviews are updated every 24 hours.', true)
       }
       else {
         console.log(res.error);
-        alert("An unknown error occured, please try again.", "danger");
+        //alert("An unknown error occured, please try again.", "danger");
```
Comment (khemritolya) [inline client/src/ui/Form.jsx:271]: here also

File: client/src/ui/Form.jsx [hunk 3]
```diff
@@ -263,7 +280,8 @@ export default class Form extends Component {
 
   submitError() {
     this.hide();
-    alert("You may only submit a review with a @cornell.edu email address, please try again.", "danger");
+    //alert("You may only submit a review with a @cornell.edu email address, please try again.", "danger");
```
Comment (khemritolya) [inline client/src/ui/Form.jsx:283]: ditto.

**Review by khemritolya — COMMENTED** [review]
There are a couple of small issues that I have, see the comments. Seems good otherwise.

**Review by khemritolya — APPROVED** [review]
Looks good!

**Pull request merged**